engine/consensus/matching/engine.go

```diff
@@ -470,42 +460,42 @@ func (e *Engine) checkSealing() {
 
 	defer e.isCheckingSealing.Store(false)
 
-	e.checkingSealing()
+	err := e.checkingSealing()
+	if err != nil {
```

Comment:
are we sure we want to crash on all errors?

Reply:
My preference would be to crash, because if something goes wrong with sealing we anyway have a problem.